modules/table/src/integrationTest/java/org/apache/ignite/internal/table/ItColocationTest.java

```diff
@@ -220,7 +220,8 @@ public CompletableFuture<Void> finish(
                     HybridTimestampTracker observableTimestampTracker,
                     TablePartitionId commitPartition,
                     boolean commitIntent,
-                    Map<TablePartitionId, IgniteBiTuple<ClusterNode, Long>> enlistedGroups,
+                    Map<ReplicationGroupId, IgniteBiTuple<ClusterNode, Long>> enlistedGroups,
```

Comment:
We may miss that. Meaning that all(?) finish users currently propagate TablePartitionId's.